scylla/tests/test_unit.py

```diff
@@ -0,0 +1,113 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from copy import deepcopy
+
+import pytest
+from six import PY2
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.base.errors import CheckException, ConfigurationError
+from datadog_checks.scylla import ScyllaCheck
+
+from .common import (
+    FLAKY_METRICS,
+    INSTANCE_ADDITIONAL_GROUPS,
+    INSTANCE_ADDITIONAL_METRICS_V2,
+    INSTANCE_DEFAULT_GROUPS,
+    INSTANCE_DEFAULT_METRICS_V2,
+    bucket_metrics,
+    get_metrics,
+    transform_metrics_omv2,
+)
+
+
+@pytest.mark.skipif(PY2, reason='OpenMetrics V2 is only available with Python 3')
+@pytest.mark.unit
```

Comment:
here and elsewhere: you don't need explicit markers if you name your files right. See [this reference](https://datadoghq.atlassian.net/wiki/spaces/AI/pages/3035497240/Team+Testing+Guidelines#Overall-Test-Structure)

Reply:
I think I just continued to mark as they were in the previous iteration of the tests but the naming convention for the files follow the pattern mentioned in the documentation you provided.